import.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#json and jsonnet files
+mkdir govuk_content_schemas
+cp -r ../govuk-content-schemas/formats ./govuk_content_schemas/formats
+cp -r ../govuk-content-schemas/dist ./govuk_content_schemas/dist
+git add ./govuk_content_schemas/
+git commit -m "Add dist and formats"
+
+#examples
+cp -r ../govuk-content-schemas/examples ./govuk_content_schemas/examples
+git add ./govuk_content_schemas/
+git commit -m "Add examples"
+
+#docs
+cp -r ../govuk-content-schemas/docs ./docs/govuk_content_schemas
+git add ./govuk_content_schemas/
+git commit -m "Add docs for content schemas"
+
+#readme
+cp ../govuk-content-schemas/README.md ./govuk_content_schemas
+git add ./govuk_content_schemas/README.md
+git commit -m "Add readme for content schemas"
+
+#spec and lib
+cp ../govuk-content-schemas/lib/govuk_content_schemas/allowed_document_types.yml ./govuk_content_schemas/
+cp -r ../govuk-content-schemas/lib/schema_generator ./lib/
```

Comment:
Nice - it's good that all the code in govuk-content-schemas is under the same modules, makes this much cleaner than it could have been.